exec/java-exec/src/test/java/org/apache/drill/exec/pop/PopUnitTestBase.java

```diff
@@ -42,7 +46,17 @@
 
   @BeforeClass
   public static void setup() {
-    CONFIG = DrillConfig.create();
+    Properties props = new Properties();
```

Comment:
Will the config here override the command line's setup if the unit test is initiated from Intellij? What about other unit tests - do they have the same problem?